step.yml

```diff
@@ -0,0 +1,136 @@
+title: AppCenter Android Deploy
+summary: Deploy your awesome Android app to AppCenter.
+description: |-
+  Deploy an Android application to [AppCenter](https://appcenter.ms).
+
+  If an `app_id` is provided, the APK file will be uploaded to the specified app on HockeyApp. 
+  Otherwise HockeyApp will decide whether it's a new app or an update to an existing app based on the package ID.
```

Comment:
Add a note: Distribution of .aab is not supported for groups/testers.

 Add a note: -only single universal apk is supported
-https://docs.microsoft.com/en-us/appcenter/build/react-native/android/#63-building-multiple-apks